lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -101,6 +104,7 @@ int location_service_get_cell_location(const struct lte_lc_cells_info *cell_data
 	err = nrf_cloud_rest_cell_pos_get(&rest_ctx, &loc_req, &result);
 
 	modem_jwt_free(jwt.jwt_buf);
+	jwt.jwt_buf = NULL;
```

Comment:
I didn't have this change.  Is this just defensive?

Unfortunately the other changes in this PR are insufficient by themselves to fix the problem.

Reply:
> Is this just defensive?
Yep.